app/views/ontologies/browser/browse.html.haml

```diff
@@ -71,8 +71,8 @@
                               = turbo_frame_tag "count_#{key}_#{link_last_part(object["id"])}", busy: true
                               %span.show-if-loading
                                 = render LoaderComponent.new(small:true)
-          %div{data:{controller: "show-filter-count browse-filters", action: "change->show-filter-count#updateCount change->browse-filters#dispatchFilterEvent"}, id: "portals_filter_container"}
```

Comment:
do you rember why you did remove this ?

Reply:
Yes I removed it to not reload the frames of the federation input chips each time the user select one